aiida/cmdline/params/options/commands/code.py

```diff
@@ -60,25 +66,32 @@ def is_not_on_computer(ctx):
     prompt_fn=is_not_on_computer,
     type=click.Path(dir_okay=False),
     cls=InteractiveOption,
-    help=('[if --store-in-db]: relative path of the executable inside the code-folder.')
+    help=('[if --store-in-db]: Relative path of the executable inside the code-folder.')
 )
 
-LABEL = options.LABEL.clone(prompt='Label', cls=InteractiveOption, help='A label to refer to this code.')
+LABEL = options.LABEL.clone(
+    prompt='Label',
+    cls=InteractiveOption,
+    help='A label to refer to this code.' +
+    '  Uniqueness is not enforced, but is required to identify a code by its label (rather than its PK or UUID).'
```

Comment:
I'm mindful of the help text getting to long; how about just:

"This label can be used to identify the code, as long as labels are unique per computer."

Reply:
Yes - I did not find a compact way to express it quickly while going through the docs.
Perhaps:

```
A label to refer to this code.
Uniqueness of 'codelabel' and 'codelabel@computername' is not enforced, but required to identify a code by its label (rather than its PK or UUID).
```